components/base/kinematicbase/ptgKinematics.go

```diff
@@ -77,8 +78,8 @@ func wrapWithPTGKinematics(
 		return nil, err
 	}
 
-	frame, err := tpspace.NewPTGFrameFromKinematicOptions(
-		b.Name().ShortName(),
+	ptgFrame, err := tpspace.NewPTGFrameFromKinematicOptions(
+		b.Name().ShortName()+"-PTGKinematics",
```

Comment:
I think we will want this to have the same name. Collision specifications should be respected.